explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -36,6 +36,10 @@
       link: "/transactions",
       title: "Transactions",
     },
+    {
+      link: "/provisioners",
+      title: "Provisioners",
+    },
```

Comment:
Wondering if we should tweak the break-point for the layout (ie. making sure we display the full nav, instead of the hamburger, once we can comfortably fit all of the options on the screen – I feel this would be the quickest solution).

<img width="1048" alt="Screenshot 2024-10-18 at 13 16 44" src="https://github.com/user-attachments/assets/4dd04b7c-9ef8-4a0d-bfeb-61c3f027861f">

Reply:
We can shrink the gap value used between the logo and the navbar items, and I think that will fix the issue.